src/overlay/Peer.cpp

```diff
@@ -1056,14 +1056,6 @@ Peer::recvGetTxSet(StellarMessage const& msg)
         StellarMessage newMsg;
         if (txSet->isGeneralizedTxSet())
         {
-            if (mRemoteOverlayVersion <
```

Comment:
Why is this no longer necessary?

Reply:
Core's current minimum overlay version is 27, which means it won't even be able to authenticate with peers that are below that. So this check is redundant.

Reply:
Nice!